### ISSUE TITLE
Enable line, arc and point visualizations based on GeoJSON geometry type

### ISSUE THREAD

**Issue by tjukanovt** [keplergl/kepler.gl#961]
**Is your feature request related to a problem? Please describe.**
Currently if I'd would like to do a line or arc visualization, I have to have a CSV with lat & lon fields so that the lines are built _inside_ kepler. Even though I would have a perfectly valid GeoJSON in EPSG:4326 with straight lines, that can only be rendered as "polygons". For me at least this is not very logical.

**Describe the solution you'd like**
I would like to be able to render GeoJSON's with different geometries with different visualization methods (Lines as lines, points as points, polygons as polygons) and not every GeoJSON as a polygon. Also if there is a straight line geometry with only two vertices, I think that should be allowed to be rendered as arcs. 

**Describe alternatives you've considered**
The current system is pretty ok if you know your way with data (Python/SQL/QGIS), but enabling more input formats would definitely lower the barrier for newcomers. 

**Additional context**
Basically here's a dataset that I had as a GeoJSON lines, but had to transform it in to a flat CSV with start and end coordinates to achieve this visualization:
![image](https://user-images.githubusercontent.com/13899537/74935150-73e70d80-53f0-11ea-8fa9-e93a6ea6d7ae.png)

**Comment by heshan0131** [comment]
As a first step:
https://github.com/keplergl/kepler.gl/issues/1019
